packages/flutter/test/material/chip_test.dart

```diff
@@ -4729,7 +4729,12 @@ void main() {
     await tester.pumpWidget(
       wrapForChip(
         theme: theme,
-        child: const RawChip(label: Text('text'), backgroundColor: backgroundColor, shape: shape),
+        child: RawChip(
+          label: const Text('text'),
+          backgroundColor: backgroundColor,
+          shape: shape,
+          onPressed: () {},
+        ),
```

Comment:
This also introduces this breaking change. A callback needs to be provided in order for the chip to be truly `enabled`